modules/340-extended-monitoring/monitoring/prometheus-rules/extended-monitoring/application-controllers.yaml

```
@@ -39,11 +40,12 @@
     for: 5m
     labels:
       severity_level: "5"
+      tier: "cluster"
```

```suggestion
      tier: "cluster"
```
All `tier: "cluster"` have to be removed.